### ISSUE TITLE
[Table] - Sketch artifacts

### ISSUE THREAD

**Issue by GnatalieH** [department-of-veterans-affairs/vets-design-system-documentation#1041]
## Description
Create or update Sketch artifacts for Table and add to the VA Design System Sketch library if they are not already there.

## Details
Need symbol artifacts for several variations:
- Default
- Sortable
- Missing data

See [Storybook](https://design.va.gov/storybook/?path=/docs/components-va-table--default) for details.

## Tasks
- [x] Create or update Sketch artifacts for component
- [x] Add Sketch artifacts to Sketch library, if not already there
- [x] Create component page in Sketch, if necessary
- [x] Add link to Sketch component page to documentation ticket

## Acceptance Criteria
- [ ] Sketch component artifacts have been created or updated
- [ ] Sketch component page has been created, if necessary
- [ ] Link to Sketch component page has been added to component documentation ticket

**Comment by seanasewell** [comment]
@GnatalieH Ready for review

**Comment by GnatalieH** [comment]
[Here's the link to Table](https://www.sketch.com/s/610156b6-f281-4497-81f3-64454fc72156/p/D9AD4361-D6AC-4BB8-845A-7D90B5A3400D) in the Sketch library. This is what will go in the documentation on design.va.gov, once it's approved. 

@humancompanion-usds I already reviewed Sean's Sketch assets and think they're great. I couldn't think of anything else we needed to represent, other than *maybe* including a symbol override option to show the sort arrow icon with focus on it.
